libraries/botbuilder-dialogs/src/dialogContext.ts

```diff
@@ -56,15 +56,22 @@ export interface DialogState {
 export class DialogContext {
     /**
       * Creates an new instance of the [DialogContext](xref:botbuilder-dialogs.DialogContext) class.
-      * 
-      * @remarks
-      * Passing in a dialog context instance will clone the dialog context.
       * @param dialogs The dialog set for which to create the dialog context.
       * @param contextOrDC The context object for the current turn of the bot.
       * @param state The state object to use to read and write dialog state to storage.
+      * @remarks
+      * Passing in a dialog context instance will clone the dialog context.
       */
     public constructor(dialogs: DialogSet, contextOrDC: TurnContext, state: DialogState);
     public constructor(dialogs: DialogSet, contextOrDC: DialogContext, state: DialogState);
+
+    /**
+      * Creates an new instance of the [DialogContext](xref:botbuilder-dialogs.DialogContext) class.
+    * @remarks Passing in a dialog context instance will clone the dialog context.
```

Comment:
Please move the remarks under the parameters. Also, the comments are not correctly aligned.